.gitattributes

```diff
@@ -0,0 +1,15 @@
+/.circleci export-ignore
```

Comment:
Can we sort everything alphabetically? :smile:

Reply:
It is indeed sorted alphabetically, in the same way returned by `ls -la` 🤷‍♂️

Reply:
`ls -la` sorts directories first. Let's sort alphabetically here.

Reply:
With dotfiles/dotfolders first, ok for you?

Reply:
Done ✔️